### DISCUSSION TITLE
How to access `the_geom' in shapefiles directly

### DISCUSSION THREAD

**Discussion by DotIN13** [gama-platform/gama#80] [Q&A] [answered]
I am wondering how would I be able to access `the_geom` column of the shapefile directly when importing, and add it as an attribute to a specie, so that I could save the original CRS from the shapefile to simulation result files for further use. I am aware of `CRS_transform`, however it would be not as accurate and less efficient.

**Comment by ptaillandier** [comment, marked as answer]
Hi,

Unfortunately, it cannot be accessed directly. You have to use the shape
attribute, but that corresponds to the geometry already projected in the
GAMA coordinate system.

Cheers,

Patrick

**Reply by DotIN13** [reply]
Thanks a lot! I understand this. I guess I just need to do the transforms myself.
